packages/phone-number-privacy/combiner/src/bls/bls-cryptography-client.ts

```diff
@@ -12,50 +12,95 @@ function flattenSigsArray(sigs: Uint8Array[]) {
   return Uint8Array.from(sigs.reduce((a, b) => a.concat(Array.from(b)), [] as any))
 }
 export class BLSCryptographyClient {
-  private verifiedSignatures: Uint8Array[] = []
-
-  public async addSignature(
-    serviceResponse: ServicePartialSignature,
-    blindedMessage: string,
-    logger?: Logger
-  ): Promise<void> {
-    logger = logger ?? rootLogger
-    const polynomial = config.thresholdSignature.polynomial
-    const sigBuffer = Buffer.from(serviceResponse.signature, 'base64')
+  private unverifiedSignatures: ServicePartialSignature[] = []
+  private verifiedSignatures: ServicePartialSignature[] = []
+  private get totalSignatures(): number {
```

Comment:
I'm very far from a naming expect, but what do you think of naming this something closer to `allSignaturesLength` to distinguish it from `allSignatures`? It seems possibly easy to confuse the two right now

Reply:
Good point. I've renamed to `allSignaturesCount` to make it more clear.